### PR TITLE
Bump PyYAML

### PR DESCRIPTION
**Pull request by KTibow**
Bump `PyYAML==5.2` to `PyYAML==5.3.1` in `requirements/main.txt`
Files changed: https://github.com/yaml/pyyaml/compare/5.2...5.3.1

### CODE REVIEW

**Comment by KTibow** [comment]
Oh... compatibility...
Note: Use Python `≥3.5` and you should be okay with `PyYAML==5.3.1`.

**Comment by mrf345** [comment]
Hi @KTibow , I think it's about time to drop the support for `Python 3.4`. feel free to remove it from:

- [travis.yml](https://github.com/mrf345/flask_minify/blob/master/.travis.yml#L4)
- [noxfile.py](https://github.com/mrf345/flask_minify/blob/master/noxfile.py#L3)
- [setup.py](https://github.com/mrf345/flask_minify/blob/master/setup.py#L47)

**Comment by KTibow** [comment]
Okay, let's see if anything is complaining elsewhere.

**Comment by KTibow** [comment]
Getting this error with `coveralls`:
```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/bin/coveralls", line 11, in <module>
    sys.exit(wear())
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/coveralls/__init__.py", line 78, in wear
    from coveralls.control import coveralls
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/coveralls/control.py", line 2, in <module>
    from coveralls.report import CoverallsReporter
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/coveralls/report.py", line 6, in <module>
    from coverage.report import Reporter
ImportError: cannot import name 'Reporter'
```
Is this existing or because of the update?

**Comment by mrf345** [comment]
Good catch! Don't worry about it, i checked out some old builds it's been there for a while.
I thinks it has to do with `python-coverall` not supporting `coverage>=5.0` https://github.com/z4r/python-coveralls/issues/73#issuecomment-615522914

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 language: python
 python:
-  - "2.7"
```
[the remark this answers is not in the document]
Comment (KTibow) [reply .travis.yml:3]: Okay, I'll try that out.

**Comment by KTibow** [comment]
Okay, I think this should work. Just check the box that resolves the changes!
Note: In `flake8`, it says:
```
WARNING: flake8 setuptools integration is deprecated and scheduled for removal in 4.x.  For more information, see https://gitlab.com/pycqa/flake8/issues/544
```
Any plans?

**Comment by mrf345** [comment]
> Okay, I think this should work. Just check the box that resolves the changes!
> Note: In `flake8`, it says:
> 
> ```
> WARNING: flake8 setuptools integration is deprecated and scheduled for removal in 4.x.  For more information, see https://gitlab.com/pycqa/flake8/issues/544
> ```
> 
> Any plans?

Not at the moment, missed that one. Let's open an issue for it.
Feel free to take it, if you have the capacity 🙈

**Pull request merged**